src/main/java/org/candlepin/insights/controller/InventoryController.java

```diff
@@ -267,41 +270,50 @@ private void extractVirtualizationFacts(Consumer consumer, Map<String, String> p
     }
 
     public void updateInventoryForOrg(String orgId) {
-        long processed = validateConduitFactsForOrg(orgId)
-            .map(hostFacts -> {
-                inventoryService.scheduleHostUpdate(hostFacts);
-                return 1;
-            })
-            .count();
-        inventoryService.flushHostUpdates();
-        log.info("Host inventory update completed for org {}. Updates: {}", orgId, processed);
+        Iterable<List<ConduitFacts>> factsPartitions = Iterables.partition(
+            () -> validateConduitFactsForOrg(orgId), 100);
```

Comment:
Let's make the batch size configurable so that we can adjust if needed.

Reply:
Done